src/org/usfirst/frc/team2399/robot/commands/DriveAnglePAdjustment.java

```diff
@@ -0,0 +1,52 @@
+package org.usfirst.frc.team2399.robot.commands;
+
+import org.usfirst.frc.team2399.robot.Robot;
+import org.usfirst.frc.team2399.robot.subsystems.DriveTrain;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class DriveAnglePAdjustment extends Command {
+	
+	private DriveTrain driveTrain = Robot.driveTrain;
+	private boolean isIncrementing;
+	
+
+    public DriveAnglePAdjustment(boolean isIncrementing) {
+    	requires(driveTrain);
+    	setInterruptible(true);
+    	this.isIncrementing = isIncrementing;
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	if(isIncrementing == true)
```

Comment:
Line 29 can be simpler! (I will leave the simplification as an exercise to the reader)